platform-includes/getting-started-config/javascript.nuxt.mdx

```diff
@@ -0,0 +1,64 @@
+To fully customize the SDK initialization, you can manually initialize the SDK for the client, server, or both.
+At build time, the Sentry Nuxt Module looks for the following two files:
+
+- Client-Side: `sentry.client.config.(js|ts)` in the root containing `Sentry.init`
+- Server-Side: `instrument.server.mjs` in the `public` directory containing `Sentry.init`
+
+In these files, you can specify the full range of <PlatformLink to="/configuration/options">Sentry SDK options</PlatformLink>.
+
+
+### Nuxt Module Setup
+
+Add the Sentry Nuxt Module to your `nuxt.config.ts` file:
+
+```javascript {filename:nuxt.config.ts}
+export default defineNuxtConfig({
+  modules: ["@sentry/nuxt/module"]
+});
+```
+
+Adding this module enables the Sentry SDK in your Nuxt application. The Sentry Nuxt Module will then automatically look for the Sentry configuration files and initialize the SDK accordingly.
+
+
+### Client-side Setup
+
+Add a `sentry.client.config.(js|ts)` file to the root of your project (this is probably the same level as the `package.json`). In this file, import and initialize Sentry, specifying any SDK options for the client:
+
+<SignInNote />
+
+```javascript {filename:sentry.client.config.(js|ts)}
+import * as Sentry from '@sentry/nuxt';
+
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+
+  // We recommend adjusting this value in production, or using tracesSampler
+  // for finer control
+  tracesSampleRate: 1.0
+});
+```
+
+### Server-side Setup
+
+1. Add an `instrument.server.mjs` file to your `public` folder. In this file, import and initialize Sentry, specifying any SDK options for the server:
+
+<SignInNote />
+
+```javascript {filename:public/instrument.server.mjs}
+import * as Sentry from '@sentry/nuxt';
+
+Sentry.init({
+    dsn: "___PUBLIC_DSN___"
+});
+```
+
+2. Add an import flag to the `NODE_OPTIONS` of your `nuxt preview` in `package.json`, so the file loads before any
+other imports:
```

Comment:
m: I'd probably word this a bit differently because it's not just about their preview command but also where people actually deploy their app. 

Something along the lines of
```suggestion
2. Add an `--import` flag to the `NODE_OPTIONS` environment variable wherever you run your application. For example, for local previews, adjust your `package.json` script:
```

+ below the code snippet we could add a sentence like

"Make sure to also add this environment variable in your deployment environment, for example on Vercel or Netlify"